### ISSUE TITLE
Request to unban two accounts.

### ISSUE THREAD

**Issue by Heimdel57** [dalaranwow/dalaran-wow#5980]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Hi, would you please unban account names sherndy and sherndy1?  I spoke to the owner of these accounts at length, and he assured me he has learned his lesson on his behavior and will not have a repeat incident in the future.  He also agreed to hear any special concessions he must make in order to have the unbanning enacted.

People may be put off by his antics, but he's actually a decent guy with a heart of gold, if you can get past his jester-like behavior.  He's a bit of a wildcard, but he means well.

Thank you very much for taking the time to read my request.  I appreciate it greatly.

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
Hello @Heimdel57 
This is the bug tracker which only lists in-game glitches or server side issues (such as problems with quests, NPCs, mechanics...)
The proper place to ask us for requests would be to send us an e-mail to 'dalaran.community@gmail.com' or to make an in-game ticket.

Thank you in advance for your understanding!
Closing.
